src/modules/settings/youtube/DropdownButton.module.css

```diff
@@ -0,0 +1,38 @@
+.button {
+  width: 290px;
```

Comment:
why are we setting width and height of this? width and height should really only be set for fixed dimension containers or images/icons. button height ideally is derived based on its contents + padding + margin